nextflow.config

```diff
@@ -14,6 +14,35 @@ process {
             device = '0'
         }
     }
+    
+    withName('ProcessRepo') {
+        container = 'bitnami/git'
+    }
+    
+    withName('RunAlmanack') {
+        container = 'community.wave.seqera.io/library/pip_almanack:34e622e308f4a975'
+    }
+    
+    withName('GenerateReport') {
+        container = 'ubuntu:jammy'
+    }
 }
 
 workDir = 'work'
+
+profiles {
+    tower {
```

Comment:
Is there now any difference between the tower profile and the default process params? If not then can just leave as default and require to be run in docker.